NetScaler/Public/Disable-NSLBServer.ps1

```diff
@@ -64,6 +67,8 @@ function Disable-NSLBServer {
         [parameter(Mandatory,ValueFromPipeline = $true, ValueFromPipelineByPropertyName)]
         [string[]]$Name = (Read-Host -Prompt 'LB server name'),
         
+        [int]$Delay,
```

Comment:
It looks like the `-Graceful` and `-Delay` parameters should be mutually exclusive, e.g. use one or the other? If this is the case, we should introduce the `Delay` in a separate parameter set.

Reply:
I have updated the code to use parameter sets that makes the Graceful and Delay options mutually exclusive.